data_validation/cli_tools.py

```diff
@@ -368,16 +368,28 @@ def _configure_validation_config_parser(subparsers):
 
 def _configure_connection_parser(subparsers):
     """Configure the Parser for Connection Management."""
+
     connection_parser = subparsers.add_parser(
         "connections", help="Manage & Store connections to your Databases"
     )
     connect_subparsers = connection_parser.add_subparsers(dest="connect_cmd")
     _ = connect_subparsers.add_parser("list", help="List your connections")
-
     add_parser = connect_subparsers.add_parser("add", help="Store a new connection")
     add_parser.add_argument(
         "--connection-name", "-c", help="Name of connection used as reference"
     )
+    add_parser.add_argument(
+        "--secret-manger-type",
```

Comment:
There's a type here. "manger" should be "manager".
This has propagated throughout the changes, if you search for "manger" throughout all changes there are 90 occurrences that are missing an "a".

Reply:
fixed the typo to manager instead of manger